requirements.txt

```diff
@@ -12,3 +12,5 @@ quaternion==3.5.2.post4
 # scipy==1.10.1
 smbus==1.1.post2
 uptime==3.0.1
+adafruit-circuitpython-lps2x
```

Comment:
These requirements were added for the dew service. They should not be added here.